studies/forms.py

```diff
@@ -62,53 +79,24 @@ class Meta:
             'short_description': 'Give your study a description here.',
             'long_description': 'Explain the purpose of your study here.',
             'contact_info': 'This should give the name of the PI for your study, and an email address where the PI or study staff can be reached with questions. Format: PIs Name (contact: youremail@lab.edu)',
-            'criteria': 'Text shown to families - this is not used to actually verify eligibility.'
+            'criteria': 'Text shown to families - this is not used to actually verify eligibility.',
+            'study_type': '''<p>After selecting a study type above, you'll be asked 
+            to fill out some study type metadata as well. This metadata is unique to the 
+            study type, and provides important configurations for building your study.</p>
+            <p>If you're not sure what to enter here, just leave the defaults (you can change this later).</p>
+            <p>For more information on study types and their metadata, please
+            <a href="https://lookit.readthedocs.io/en/develop/experimenter.html#editing-study-type">see the documentation.</a></p>'''
```

Comment:
Let's take the inline HTML out of class members - either hardcode it in the view or create a partial that knows how to deal with (iterates over) these segments of help text

Reply:
Actually forget what I said - this is small potatoes and apparently there's precedent for doing it this way https://docs.djangoproject.com/en/2.1/ref/models/fields/#help-text